### DISCUSSION TITLE
Installs not Working.

### DISCUSSION THREAD

**Discussion by Amez91** [Guilouz/Creality-Helper-Script-Wiki#604] [Issues]
hi, when i try to install Timelapse and a few others from the installer helper i get errors.
I am unsure what i am doing wrong due to being new to klipper.
Any help would be appreciated. Thank you 
The picture is the example for time-lapse, it also happens on the Klipper mesh one and camera settings control.
I have installed octoeverywhere successfully tho.
![Image 08-06-2024 at 16 09](https://github.com/Guilouz/Creality-Helper-Script-Wiki/assets/172129415/f15ba02c-be47-4349-b42e-9923b86e2bcf)

Edited to mention i am running it on an ender 3 v2 with a nebula pad.

**Comment by Guilouz** [comment]
I think you do not use latest script version.

**Reply by Amez91** [reply]
When I update to the latest script version the ./installer.sh does not bring up the installer? 
Sorry I am a little new to this and have fired my brain learning all so far!
